playbooks/group_vars/all.yml

```diff
@@ -52,7 +47,7 @@ enable_docket: false
 enable_bro: true
 enable_suricata: true
 enable_snort: false
-enable_pulledpork: false
+enable_suricata_update: false
```

Comment:
We need to enable this by default, otherwise the rules will never deploy on install.